db/migrate/20180917082341_add_conversion_host_limits.rb

```diff
@@ -0,0 +1,11 @@
+class AddConversionHostLimitsColumns < ActiveRecord::Migration[5.0]
+  def change
+    change_table :conversion_hosts do |t|
+      t.string :limits_max_concurrent_transformations
```

Comment:
I'm not sure about the naming and plualization here, also `limit` + `max` seems redundant. Wouldn't this be clearer as `concurrent_transformation_limit`?